geometry_msgs/PointStamped position
float64 rate